modules/installation-guide/partials/con_persistent-volume-configuration.adoc

```diff
@@ -5,95 +5,121 @@
-In this strategy, all {prod-short} workspaces use the same PVC. When the user runs one workspace, it only binds to one node in the cluster at a time.
+|===
 
-The corresponding containers volume mounts link to a common volume, and sub-paths are prefixed with `_<workspace-ID>_` or `__<original-PVC-name>__`. For more details, see xref:how-subpaths-are-used-in-pvcs_{context}[].
+Persistent volumes (PVs) access mode::
+The nature of the PV determines the available access mode: `ReadWriteMany` or `ReadWriteOnce`.
+See link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[Kubernetes documentation - access mode].
+For example, link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html[Amazon EBS] supports only the `ReadWriteOnce` access mode.
 
-The {prod-short} Volume name is identical to the name of the user-defined PVC. It means that if a machine is configured to use a {prod-short} volume with the same name as the user-defined
-PVC has, they will use the same shared folder in the common PVC.
 
-When a workspace is deleted, a corresponding subdirectory (`$\{ws-id}`) is deleted in the PV directory.
+[id="the-common-storage-strategy_{context}"]
+== The `common` storage strategy
 
-NOTE: The common PVC will be removed when a user's workspaces are all deleted. The PVC will be re-created when a non-ephemeral workspace is started.
+This is the default storage strategy.
+For each user, all workspaces use the same PV for the default data storage.
 
-.Restrictions on using the `common` PVC strategy
+When the user starts a first non-ephemeral workspace, the workspace engine creates a common PV.
 
-When the `common` strategy is used, and a workspace PVC access mode is ReadWriteOnce (RWO), only one node can simultaneously use the PVC.
+When the user starts another non-ephemeral workspace, the workspace engine uses the same common PV.
+The workspace only binds simultaneously to one node in the {orch-name} cluster.
+The workspace engine ignores user-defined volumes. 
+The workspace engine replaces volumes related to user-defined volumes with a subPath in the common volume. 
+SubPaths have a `_<workspace-ID>_` or `__<original-volume-name>__` prefix.
+See xref:how-subpaths-are-used-in-persistent-volumes_{context}[].
+The {prod-short} volume name is identical to the name of the user-defined PV.
+Therefore, if a workspace container uses a {prod-short} volume with the same name as the user-defined PV, they will use the same shared folder in the common PV.
 
-If there are several nodes, you can use the `common` strategy, but:
+When the user deletes a workspace, the workspace engine deletes the corresponding subdirectory (`$\{ws-id}`) in the PV directory.
 
-* The workspace PVC access mode must be reconfigured to `ReadWriteMany` (RWM), so multiple nodes can use this PVC simultaneously.
+When the user deletes the last workspace, the workspace engine removes the common PV. 
```

Comment:
Suggestion: Group the related items in a list and have these listed as bullet points. Seeing all these different possibilities and reading them in one paragraph is difficult as a user/reader to understand and process the content in your mind.

I hope it is not just me. See what is helpful from the user/reader's perspective and do the needful.